thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/controller.js

```diff
@@ -76,6 +90,16 @@ export default Controller.extend({
     resolution: 'anomalyFeedback'
   },
 
+  /**
+   * Conditional formatting for tuning fields
+   */
+  isTuneAmountPercent: computed(
+    'selectedSeverityOption',
+    function() {
+      return this.get('selectedSeverityOption') !== 'Absolute Value of Change';
```

Comment:
Also, missing `@type`. Since you're using the new import method with `computed`, you can also apply that to `this.get` by using `get(this, ...)`. I think you can use `Ember.computed.not` here: http://www.emberjs.com.cn/api/classes/Ember.computed.html#method_not

Reply:
for consistency, not refactoring the tuning flow to new `get` syntax yet.